src/components/CloudHookForm.vue

```diff
@@ -165,6 +165,10 @@ export default {
       return allHooks.filter(
         t => (t.requiresCloud && this.isCloud) || !t.requiresCloud
       )
+    },
+    projectNamePrefix() {
+      const prefixedName = item => item.name + ' (' + item.project.name + ')'
```

Comment:
```suggestion
      const prefixedName = item => `item.name (${item.project.name})`
```
I'd suggest interpolating this string instead, it's a little easier to maintain. You can also directly return this function instead of allocating a variable first